runtime/src/lib.rs

```diff
@@ -193,134 +204,161 @@ fn handle_command(drivers: &mut Drivers) -> CaliptraResult<MboxStatusE> {
         req_packet.payload().len()
     );
 
-    // Handle the request and generate the response
-    let mut resp = match CommandId::from(req_packet.cmd) {
+    // The mailbox is much larger than the request and response buffers.
+    // We can use the second half of the mailbox to stage the response, which we
+    // can copy to the first half of the mailbox before sending it. This saves us
+    // ~10 KB of stack space.
+    let mbox = Mailbox::raw_mailbox_contents_mut();
+    let mbox_len = mbox.len();
+    let (_, resp) = mbox.split_at_mut(mbox_len / 2);
+    let resp = &mut resp[..MAX_RESP_SIZE];
```

Comment:
Maybe it's worth some kind of validation that requests don't overlap this value? One one way would be to statically compare sizes, another is to dynamically compare to payload().len().

Reply:
Will resolve in separate PR